packages/react-server-cli/src/coreCssMiddleware.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = function(pathToStatic) {
```

Comment:
> Already doing that browser-side...

Yeah, although I _think_ we could remove it as a browser-side global now (i.e. get rid of `window.__reactServerBase`; it shouldn't be used any more.) We still have to leave the method that assigns it to `__webpack_public_path__`, though, or else the webpacked code freaks out that it's on a different server than the base HTML.

I hear what you are saying, but these two middlewares are not intended to be externally available, so the only client should be the transpiled routes file, which calls the constructor with the path to the static files. That seems better to me than adding a global that others might start depending on; globals have a way of becoming impossible to modify/remove because it's so hard to find their dependents. I also worry about having a global that's defined in `react-server-cli`; if app code depends on the global, the app won't be portable between `react-server-cli` and raw `react-server`.

So: I'd prefer this not depend on a global, but if it's what's needed to get the code pulled, I can totally do so. ;)